bylaws/001-mission-and-structure.md

```diff
@@ -24,59 +24,63 @@ A Member Project may not have more than one Project Representative at a time. No
 
 ## Secretaries
 
-The primary responsibility of the FIG Secretary is to serve as an impartial administrator of the FIG. They serve at the member projects pleasure but act as independent and impartial adjudicators. FIG Secretaries will also represent the FIG as a whole to the general community and public on matters relating to the FIG's activities.
+The primary responsibility of the FIG Secretary is to serve as an impartial administrator of the FIG. They serve at the member projects and core committee's pleasure but act as independent and impartial adjudicators. FIG Secretaries also represent the FIG as a whole to the general community and public on matters relating to the FIG's activities.
 
 Whilst there are a number of defined functions below but they should also perform other duties as required. As the role has continuity (in that there will always be a post holder) and redundancy (There are a number of post holders in case of the absence of one) it can be ensured that responsibilities assigned to the secretary will always be completed and therefore administrative responsibilities such as vote management should always be assigned to the FIG Secretary.
 
 All FIG Secretaries are expected to remain impartial and professional when acting in an official capacity as Secretary and should also remain aware that even when not acting in an official capacity, their actions reflect back on the PHP FIG and the FIG must not be brought into disrepute.
 
-In order to ensure impartiality, Secretaries may not concurrently serve as a Project Representative, Core Committee Member, or Editor of a PSR. They may, however, be a member of a Working Group.
+In order to ensure impartiality, Secretaries may not concurrently serve as a Project Representative, Core Committee Member, or Editor of a PSR. They may, however, be a member of a Working Group. They must declare all conflicts of interest (for example being on the core team of member projects or involvement in PSR working groups)
 
 ### Defined functions
 
 There are a number of defined functions that the FIG Secretaries are expected to complete but they may also, within their remit defined in the above overarching role, perform other duties as necessary.
 
-* Keeping the website, Twitter and other marketing mediums up-to-date
-* GitHub organization administration
-* Tallying votes
-* Tracking member project activity, and marking projects as inactive
+* Managing the website
+* GitHub organization and repository administration
+* Tallying votes and managing the voting system
+* Tracking member project activity
 * Ensuring bylaws are being followed
-* Clarifying any interpretation of bylaw text (If there is lack of consensus between FIG Secretaries then it must be put to a full vote)
+* Clarifying any interpretation of bylaw text (If there is lack of consensus between FIG Secretaries then it must be put to a full vote); subject to a lack of objection from Core Committee members and Project Representatives**
```

Comment:
The wording here, between the semicolon and the parenthetical, is grossly confusing.  I don't even know how to interpret it.